addOns/ascanrulesAlpha/src/main/java/org/zaproxy/zap/extension/ascanrulesAlpha/ExampleFileActiveScanner.java

```diff
@@ -47,7 +47,7 @@
     /** Prefix for internationalized messages used by this rule */
     private static final String MESSAGE_PREFIX = "ascanalpha.examplefile.";
 
-    private static final String exampleAscanFile = "xml/example-ascan-file.txt";
+    private static final String exampleAscanFile = "txt/example-ascan-file.txt";
```

Comment:
Being a good example :) I'd suggest using the `addOnData` dir, IMO we should move away from format/type dir names.
The corresponding passive scan rule should have the same changes for consistency.